src/templates/_components/ads/sidebar.twig

```diff
@@ -0,0 +1,26 @@
+{% import 'translations/_components/_forms/button' as button %}
+
+{% if ads is not empty %}
+	{% set heading = ads.heading %}
+	{% set content = ads.content %}
+{% endif %}
+
+<div id="sidebar-ad" class="meta">
+	<div class="ad-container sidebar-ad">
+		<div class="field">
+			<div class="heading">
+				<label> {{ heading }} </label>
+			</div>
+
+			<div class="input ltr">
+				<label> {{ content }} </label>
+			</div>
+			<div class="flex" style="justify-content: end;">
```

Comment:
As mentioned here: https://github.com/AcclaroInc/craft-translations/pull/527#issuecomment-2422857319

The button should be aligned left

Reply:
done